docs/HowToGuides.md

```diff
@@ -1,5 +1,78 @@
 # How To ...
 
+### Create a multi-step PR
+
+> Ask a programmer to review 10 lines of code, he'll find 10 issues. Ask him to do 500 lines and he'll say it 
+> looks good. _[[source](https://twitter.com/girayozil/status/306836785739210752)]_
+
+Big PRs (i.e. PRs touching more than 100 lines) are hard to review. Using the _multi-step PR_ format allows the 
+review to be done incrementally.
+
+A multi-step PR consists a series of commits corresponding to a sequence of logical steps, 
+each step advancing the code base towards the end goal of the PR. The build must pass at each step. 
```

Comment:
Also, we may wish to show some hypothetical examples on what is not a good multi-step PR, in order to aid the reader's understanding. But let's leave it for later since that takes some effort to do.